lib/bundle/brewfile.rb

```diff
@@ -5,26 +5,37 @@ module Brewfile
     module_function
 
     def path(dash_writes_to_stdout: false)
-      if ARGV.include?("--global")
-        Pathname.new("#{ENV["HOME"]}/.Brewfile")
-      else
-        filename = ARGV.value("file")
-        if filename == "-"
-          filename = if dash_writes_to_stdout
-            "/dev/stdout"
+      env_bundle_file = ENV["HOMEBREW_BUNDLE_FILE"]
+
+      filename =
+          if ARGV.include?("--global")
+            raise "'HOMEBREW_BUNDLE_FILE' can not be specified with '--global'" if env_bundle_file.present?
+            "#{ENV["HOME"]}/.Brewfile"
+          elsif ARGV.include?("--file")
```

Comment:
When using `brew bundle --file=path/to/Brewfile`, `ARGV` will be `["--file=path/to/Brewfile"]`. So, the check of `ARGV.include?("--file")` is wrong.

Reply:
Thanks for pointing this out. It seems like this might need to be adjusted in the mainline now that the PR has been closed.